README.md

```diff
@@ -144,7 +144,19 @@ The next time you invoke a documentation build with the "Build Documentation"
 button in Xcode's Product menu, your custom `docc` will be used for the build.
 You can confirm that your custom `docc` is being used by opening the latest build
 log in Xcode's report navigator and expanding the "Compile documentation" step.
-  
+
+### Invoking `docc` from Swift Package Manager
+
+You can also test a locally built version of Swift-DocC using Swift Package Manager
+CLI. SwiftPM will try to read `DOCC_EXEC` environment variable value, and use
+the path you provded if it's set.
+
+  1. In your test `Package.swift`, add a dependency on [`Swift-DocC Plugin`](https://github.com/apple/swift-docc-plugin).
+  2. Set `DOCC_EXEC` environment variable and run documentation generation
+     command like this:
+
+        `DOCC_EXEC=/path/to/docc swift package generate-documentation`
```

Comment:
What do you think of updating this to a code block to match the existing formatting? Currently it stands out from the others and is more difficult to copy/paste.

```suggestion
        ```bash
        DOCC_EXEC=/path/to/docc swift package generate-documentation
        ```
```

<img width="1112" alt="Screenshot 2023-07-07 at 10 39 25 AM" src="https://github.com/apple/swift-docc/assets/6750147/32c64d53-06f3-42bc-9b6b-69da7ad9d91f">